ma/maSnapper.h

```diff
@@ -27,20 +30,34 @@ class FirstProblemPlane;
 class Snapper
 {
   public:
-    Snapper(Adapt* a, Tag* st, bool is);
+    int numFailed = 0;
+    int numSnapped = 0;
+    int numCollapseToVtx = 0;
+    int numCollapse = 0;
+    int numSwap = 0;
+    int numSplitCollapse = 0;
+
+    Snapper(Adapt* a, Tag* st);
+    ~Snapper();
     void setVert(Entity* v);
     Entity* getVert();
     bool requestLocality(apf::CavityOp* o);
     bool trySimpleSnap();
     bool run();
-    bool dug;
-    bool moved;
   private:
-    Adapt* adapter;
+    Adapt* adapt;
+    Mesh* mesh;
```

Comment:
IIRC, the adapt object contains a pointer to the mesh.  I'm guessing this was added for convenience.  Can/should this pointer be `const` and set via the class constructor initializer list?